data-model/src/main/avro/RawEventLog.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventRecordsProtocol {
```

Comment:
nit: Generally the name of the file is matching with the name of the message defined in the file. So, shall we rename the file as `LogEvents.avdl` and `LogEventsProtocol`?

Reply:
Is there any convention we follow for this, can't we use tooling?